### PR TITLE
Add workflow for deploying Bitfinity Reth to GHCR

### PR DESCRIPTION
**Pull request by itsyaasir**

### CODE REVIEW

**Comment by ufoscout** [comment]
@itsyaasir What about the workflow to publish the docker image as github package? Will it be part of another PR?

**Comment by itsyaasir** [comment]
> @itsyaasir What about the workflow to publish the docker image as github package? Will it be part of another PR?

This is what the PR does

**Comment by ufoscout** [comment]
> This is what the PR does

Ah! So it is not "gcr" but "gHcr" :)

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: 'Deploy Bitfinity Reth to GCR'
```
Comment (ufoscout) [inline .github/workflows/publish.yml:1]: typo: GCR -> GHCR

**Review by ufoscout — CHANGES_REQUESTED** [review]
Please add instructions to use the docker image

**Review by GraDKh — APPROVED** [review]

**Review by ufoscout — APPROVED** [review]

**Pull request merged**